companies/zucchetti.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "zucchetti",
+    "relevant-countries": [
+        "it",
+        "us",
+        "spa",
+        "br",
+        "fr"
+    ],
+    "categories": [
+        "it",
+        "commerce"
+    ],
+    "name": "Zucchetti",
+    "address": "Via Solferino 1, 26900 Lodi - Italia",
```

Comment:
To keep our records consistent, we always delimit the address parts by newlines (see [our data format guidelines](https://github.com/datenanfragen/data#data-format-guidelines-and-resources-for-company-records)).